packages/components/pages/f-self-exclusion/src/components/SelfExclusion.vue

```diff
@@ -172,21 +172,24 @@ import tenantConfigs from '../tenants';
 
 export default {
     name: 'SelfExclusion',
+
     components: {
         FCard,
         FAlert,
         FButton,
         FFormField
     },
+
     mixins: [VueGlobalisationMixin],
+
     props: {
         locale: {
             type: String,
             default: 'en-AU'
         },
         authToken: {
             type: String,
-            required: true
+            default: null
```

Comment:
Two question about this.
1.Do you not want to default this to an empty string? 
2. Isn't `authToken` required to make the call? So should required remain in place?

Reply:
For both, during testing with CoreWeb, the requests (that require the authToken) did not work with `required: true`. We consulted with the CoreWeb team and they advised to add `default: null`.

Reply:
OK. I think i'd need to look at why to answer properly but the reason i ask is because if you use this component elsewhere and you don't pass in a value for `authToken` (someone might not know this) the call will fail surely? Also you might want to update/fill out the prop table with those details now.
https://github.com/justeattakeaway/fozzie-components/blob/master/packages/components/pages/f-self-exclusion/README.md#props

Reply:
We tested again and there seems to be no issue now with adding required: true.

Reply:
Yes if we can update in another PR it would be great, so we can create the PR for CoreWeb.